core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -98,8 +107,8 @@ class Validator private (private val _name : String, val startStep : Step, val c
 
 
   private val platformMBeanServer = ManagementFactory.getPlatformMBeanServer()
-  private val objectName = new ObjectName("\"com.rackspace.com.papi.components.checker\":type=\"Validator\",scope=\""+
-                                 name+"\",name=\"checker\"")
+  private val objectName = new ObjectName("com.rackspace.com.papi.components.checker:type=Validator,scope="+
+                                 name+",name=checker")
```

Comment:
They are public facing, but only used in the lookup of the metrics in JMX. That said, there may be someone that is using them. At the same time this is a major version with other breaking changes too.